src/homeworks/homework-1.ts

```diff
@@ -1,115 +1,123 @@
 class School {
-  directions = [];
+  directions: string[] = [];
 
-  addDirection(direction) {
+  addDirection(direction: string) {
     this.directions.push(direction);
   }
 }
 
 class Direction {
-  levels = [];
+  private _name: string;
+  levels: number[] = [];
 
-  get name() {
+  get name(): string {
     return this._name;
   }
 
-  constructor(name) {
+  constructor(name: string) {
     this._name = name;
   }
 
-  addLevel(level) {
+  addLevel(level: number) {
```

Comment:
Please ensure that you include a type annotation for the function's return value.